apps/dashboard-service/src/dashboard/dto/tree-restoration-goal.dto.ts

```diff
@@ -0,0 +1,74 @@
+import { JsonApiDto } from "@terramatch-microservices/common/decorators";
+import { ApiProperty } from "@nestjs/swagger";
+import { populateDto } from "@terramatch-microservices/common/dto/json-api-attributes";
+
+class TreeRestorationData {
+  @ApiProperty({
+    description: "Due date for this restoration data",
+    type: String,
+    example: "2024-01-01T00:00:00.000000Z"
+  })
+  dueDate: string;
+
+  @ApiProperty({
+    description: "Number of tree species for this period",
+    type: Number
+  })
+  treeSpeciesAmount: number;
+
+  @ApiProperty({
+    description: "Percentage of tree species for this period",
+    type: Number
+  })
+  treeSpeciesPercentage: number;
+}
+
+@JsonApiDto({ type: "treeRestorationGoals" })
+export class TreeRestorationGoalDto {
+  constructor(data: TreeRestorationGoalDto) {
+    populateDto<TreeRestorationGoalDto>(this, data);
+  }
+
+  @ApiProperty({
+    description: "Total number of trees grown goal for for-profit organizations",
+    type: Number
+  })
+  forProfitTreeCount: number;
+
+  @ApiProperty({
+    description: "Total number of trees grown goal for non-profit organizations",
+    type: Number
+  })
+  nonProfitTreeCount: number;
+
+  @ApiProperty({
+    description: "Total trees grown goal across all organizations",
+    type: Number
```

Comment:
🎨 It doesn't hurt anything, but FYI the `type` isn't needed unless the property is nullable (uses `: number | null`). In cases where the type is concrete, the swagger generation layer can correctly determine the type without it.